editor/modes/visual-editor/block.js

```diff
@@ -104,6 +110,20 @@ class VisualEditorBlock extends wp.element.Component {
 		if ( 27 /* Escape */ === event.keyCode ) {
 			this.props.onDeselect();
 		}
+
+		if ( metaKeyPressed( event ) ) {
+			this.metaCount++;
+		} else {
+			this.metaCount = 0;
+		}
+	}
+
+	handleKeyUp() {
+		if ( this.metaCount === 1 ) {
```

Comment:
Maybe a bit odd to expect, but if I hold down a non-meta character (say, `c`), then press a meta character, when releasing, `metaCount` is `=== 1`.